src/components/Dropdown.js

```diff
@@ -277,6 +283,7 @@ Dropdown.propTypes = {
   positionX: PropTypes.string,
   arrowStyles: PropTypes.shape(),
   onRef: PropTypes.func,
+  onMouseLeave: PropTypes.func,
```

Comment:
I would change that prop to `closeOnMouseLeave` as well and just run `close()` function on mouseLeave event if true. That will make both `DropdownWithToggler` and `Dropdown` components more consistent ;)

Reply:
But we do already pass `close()` function to this component and the same way we have `closeOnItemClick` prop I will expect it to have `closeOnMouseLeave` I think :/